src/pages/iou/MoneyRequestDatePage.js

```diff
@@ -37,16 +38,20 @@ const propTypes = {
             threadReportID: PropTypes.string,
         }),
     }).isRequired,
+
+    /** The current tab we have navigated to in the request modal. String that corresponds to the request type. */
+    selectedTab: PropTypes.oneOf([CONST.TAB.DISTANCE, CONST.TAB.MANUAL, CONST.TAB.SCAN]).isRequired,
```

Comment:
NAB: This `CONST` could be refactored a little to remove `RECEIPT_TAB_ID` and then this could use `_.values(CONST.TAB)`

Reply:
might do as a cleanup issue 🙇